seven/Counter.java

```diff
@@ -0,0 +1,25 @@
+package seven;
+
+import java.util.concurrent.TimeUnit;
+public class Counter {
+ void printCount()
+ {
+	 for(int i=1;i<=100;i++)
+	 {  
+		 try
+		 {  
+			TimeUnit.MILLISECONDS.sleep(1000);
+			 System.out.println("..............."+i+"...............");
+			 
+			 if(i%10==0)
+				 System.out.println(">>>>>>>>>>Ten seconds are over<<<<<<<<<<");		 
+		 }
+		 catch(Exception e)
+		 {
+			 
```

Comment:
@Hariharan-s005 Avoid having an empty catch block, Try to have some useful information logged there, like print stack trace. (Applicable for all empty catch blocks used, change all empty catch blocks)

cc: @jenisha-roshan